foundationdb/tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{27,38}
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py27: {toxworkdir}/py27
+    py38: {toxworkdir}/py38
+dd_check_style = false
```

Comment:
Let's enable this

Reply:
If we must, than at the very end, as the last step before the merge.

Reply:
Ah, and to add *why*, since it could be useful feedback: when using the `ddev` command to run tests, it also would run the style check. When one is at the point of trying to debug a test failure, especially when new to building an integration and with a lot to figure out, about the last thing wanted is "oh, you have a couple of lines that are too many characters long!"

Reply:
Just FYI, you can run `ddev test foundationdb:py38` to only run py38 test without having to care about style test during the development phase. But no blocker to turn it on only once the implementation is fully ready.

Reply:
Ah, good to know; thanks.